### PR TITLE
Phpcs

### PR DESCRIPTION
**Pull request by ADmad**
I had to suppress sniffs related to docblocks for now as it requires lot of manual work :slightly_smiling_face:.

### CODE REVIEW

File: src/Event/ExtensionsListener.php [hunk 1]
```diff
@@ -47,7 +48,7 @@ public function implementedEvents(): array
     /**
      * Event handler.
      *
-     * @param ConstructEvent $event Event.
+     * @param \WyriHaximus\TwigView\Event\ConstructEvent $event Event.
```
Comment (WyriHaximus) [inline src/Event/ExtensionsListener.php]: Why are you using a full class here instead of using the imported at the top of the file?

Comment (ADmad) [reply src/Event/ExtensionsListener.php]: Cake's convention is to use FQCN in docblocks, it makes for non ambiguous API docs.

Comment (WyriHaximus) [reply src/Event/ExtensionsListener.php]: Check :+1:

**Comment by WyriHaximus** [comment]
THanks :tada:

**Pull request merged**